specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-02-preview/components_API.json

```diff
@@ -601,6 +601,14 @@
             "name": "IngestionMode",
             "modelAsString": true
           }
+        },
+        "DisableNonAADAuth": {
```

Comment:
Shouldn't `DisableNonAADAuth` be `DisableLocalAuth`?

Reply:
fixed